src/ec2/resources/crontab

```diff
@@ -6,5 +6,6 @@
 10 * * * * ec2-user docker run --rm 611413694531.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger:latest python /root/BridgeDownstream/src/scripts/bootstrap_trigger/bootstrap_trigger.py --file-view syn29357582 --raw-folder-id syn29300178 --glue-workflow bridge-downstream-mobile-toolbox-fmqcjv-S3ToJsonWorkflow --ssm-parameter synapse-bridgedownstream-auth --diff-s3-uri 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/fmqcjv/parquet/dataset_sharedschema_v1/' --diff-parquet-field recordid --diff-s3-uri-2 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/fmqcjv/parquet/dataset_archivemetadata_v1/' --diff-parquet-field-2 recordid --diff-file-view-field recordId --drop-duplicates
 15 * * * * ec2-user docker run --rm 611413694531.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger:latest python /root/BridgeDownstream/src/scripts/bootstrap_trigger/bootstrap_trigger.py --file-view syn30034467 --raw-folder-id syn29456518 --glue-workflow bridge-downstream-mobile-toolbox-hktrrx-S3ToJsonWorkflow --ssm-parameter synapse-bridgedownstream-auth --diff-s3-uri 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/hktrrx/parquet/dataset_sharedschema_v1/' --diff-parquet-field recordid --diff-s3-uri-2 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/hktrrx/parquet/dataset_archivemetadata_v1/' --diff-parquet-field-2 recordid --diff-file-view-field recordId --drop-duplicates
 20 * * * * ec2-user docker run --rm 611413694531.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger:latest python /root/BridgeDownstream/src/scripts/bootstrap_trigger/bootstrap_trigger.py --file-view syn30795668 --raw-folder-id syn29456954 --glue-workflow bridge-downstream-mobile-toolbox-pmbfzc-S3ToJsonWorkflow --ssm-parameter synapse-bridgedownstream-auth --diff-s3-uri 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/pmbfzc/parquet/dataset_sharedschema_v1/' --diff-parquet-field recordid --diff-s3-uri-2 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/pmbfzc/parquet/dataset_archivemetadata_v1/' --diff-parquet-field-2 recordid --diff-file-view-field recordId --drop-duplicates
-25 * * * * ec2-user docker run --rm 611413694531.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger:latest python /root/BridgeDownstream/src/scripts/bootstrap_trigger/bootstrap_trigger.py --file-view syn51121831 --raw-folder-id syn51109164 --glue-workflow bridge-downstream-mobile-toolbox-ccbcwq-S3ToJsonWorkflow --ssm-parameter synapse-bridgedownstream-auth --diff-s3-uri 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/ccbcwq/parquet/dataset_sharedschema_v1/' --diff-parquet-field recordid --diff-s3-uri-2 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/ccbcwq/parquet/dataset_archivemetadata_v1/' --diff-parquet-field-2 recordid --diff-file-view-field recordId --drop-duplicates
+# See https://sagebionetworks.jira.com/browse/ETL-432 for query rational
```

Comment:
Do you expect this query to ever change? Do you think we will ever have to reprocess all the data again?

Reply:
Got it.  Thanks - this makes more sense now!